app/views/forums/_head.html.haml

```diff
@@ -1,10 +1,17 @@
-- if object
-  - presenter = present(object)
+%nav
+  %ol.breadcrumb
```

Comment:
The breadcrumb looks quite off on mobile. It might really need it's own row entirely to be usable.

Reply:
Ah yeah, this is what I wanted to ask you about.

At the moment, the `Edit Topic` & `Subscribe` buttons are being "inlined" with the breadcrumb nav by using `.pull-right`.

This is because those buttons are defined in the individual `_show.html.haml` files; so it's a little tricky to lay-it-out with the breadcrumb nav in `_head.html.haml`.

My questions are:

1. Is it okay to move those buttons into `_head.html.haml`? There would need to be some logic to determine the passed `object` type, so I can conditionally render "Thread" buttons or "Topic" buttons.

2. Actually do you think - instead of what I've done - try to implement some more generic breadcrumb handler (might be useful for some of the other views too).

Reply:
I don't think there's much benefit to abstracting breadcrumbs. Moving the buttons is fine tough, they're not used in many places.